lib/gcloud/resource_manager/manager.rb

```diff
@@ -0,0 +1,318 @@
+      #   projects.each do |project|
+      #     puts project.project_id
+      #   end
+      #
+      # If you have a significant number of projects, you may need to paginate
+      # through them: (Gcloud::ResourceManager::Project::List)
+      #
+      #   require "gcloud"
+      #
+      #   gcloud = Gcloud.new
+      #   resource_manager = gcloud.resource_manager
+      #   projects = resource_manager.projects.all
+      #   projects.each do |project|
+      #     puts project.project_id
+      #   end
+      #
+      def projects filter = nil, options = {}
+        resp = connection.list_project list_projects_options(filter, options)
+        if resp.success?
+          Project::List.from_response resp, self
+        else
+          fail ApiError.from_response(resp)
+        end
+      end
+
+      ##
+      # Retrieves the project identified by the specified +project_id+.
+      #
+      # === Parameters
+      #
+      # +project_id+::
+      #   The ID of the project. (+String+)
+      #
+      # === Returns
+      #
+      # Gcloud::ResourceManager::Project, or +nil+ if the project does not exist
+      #
+      # === Example
+      #
+      #   require "gcloud"
+      #
+      #   gcloud = Gcloud.new
+      #   resource_manager = gcloud.resource_manager
+      #   project = resource_manager.project "tokyo-rain-123"
+      #   project.project_id #=> "tokyo-rain-123"
+      #
+      def project project_id
+        resp = connection.get_project project_id
+        if resp.success?
+          Project.from_gapi resp.data, connection
+        else
+          nil
+        end
+      end
+
+      ##
+      # Creates a project resource.
+      #
+      # Initially, the project resource is owned by its creator exclusively. The
+      # creator can later grant permission to others to read or update the
+      # project.
+      #
+      # Several APIs are activated automatically for the project, including
+      # Google Cloud Storage.
+      #
+      # === Parameters
+      #
+      # +project_id+::
+      #   The unique, user-assigned ID of the project. It must be 6 to 30
+      #   lowercase letters, digits, or hyphens. It must start with a letter.
+      #   Trailing hyphens are prohibited. (+String+)
+      # +options+::
+      #   An optional Hash for controlling additional behavior. (+Hash+)
+      # <code>options[:name]</code>::
+      #   The user-assigned name of the project. This field is optional and can
+      #   remain unset.
+      #
+      #   Allowed characters are: lowercase and uppercase letters, numbers,
+      #   hyphen, single-quote, double-quote, space, and exclamation point.
+      #   (+String+)
+      # <code>options[:labels]</code>::
+      #   The labels associated with this project.
+      #
+      #   Label keys must be between 1 and 63 characters long and must conform
+      #   to the following regular expression: [a-z]([-a-z0-9]*[a-z0-9])?.
```

Comment:
RDoc creates a link for the `[a-z]` fragment when I generate locally. Not sure how to prevent it however...